src/test/java/hudson/tasks/AntTest.java

```diff
@@ -123,6 +124,12 @@ private void verify() throws Exception {
         assertEquals(1,l.length);
         r.assertEqualBeans(l[0],new AntInstallation("myAnt","/tmp/foo", JenkinsRule.NO_PROPERTIES),"name,home");
 
+        // Verify that PATH+ANT is set.
```

Comment:
Yup, did manual testing, and I'd happily add a test in `pipeline-model-definition`.